.travis.yml

```diff
@@ -1,29 +1,45 @@
+sudo: required
+
 language: go
 
 go:
   - 1.7.x
   - 1.8.x
```

Comment:
I would add `master` – 1.9 is just around the corner with changes in `database/sql`.